st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -38,6 +38,7 @@
 import platform
```

Comment:
The config file that you're adding should provide the ability to configure a company name. The default should be "StackStorm". We''ll override it to "Plexxi". It should be used here to replace the StackStorm references with Plexxi references:

regana@Reganas-MBP ~/Development/git/plexxi_st2/st2debug/st2debug/cmd> grep StackStorm submit_debug_info.py
LOG.exception('Failed to upload tarball to StackStorm', exc_info=True)
LOG.info('Debug tarball successfully uploaded to StackStorm (name=%s)' % (tarball_name))
print('This will submit the following information to StackStorm: %s' % (submited_content))